javascript.md

```diff
@@ -1004,12 +1064,88 @@
   </p>
   </details>
 
+  -
+  <details>
+    <summary>
+      <b>A11.</b> When typescript is used, type-driven development is used as much as possible to ensure compile time safety. Compiler is always set to the strictest mode possible. Any use of `any` or other unsafe operations are commented. Spread is avoided for objects with methods.
```

Comment:
Before that, we did not have TS specific rules. Not all projects will use TS. It would be logical to somehow highlight these rules (for example, add emoji)